songs/149_Du_gamla_du_fria.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: Du gamla, du fria
+alternativeTitles:
+  - Sveriges nationalsång
```

Comment:
Perhaps also add the definite article "Nationalsången"

Reply:
Added that as well :)